modules/tinymce/src/core/test/ts/browser/EditorManagerTest.ts

```diff
@@ -87,7 +87,7 @@ UnitTest.asynctest('browser.tinymce.core.EditorManagerTest', (success, failure)
       content_css: '/project/tinymce/js/tinymce/skins/content/default',
       language: langCode,
       language_url: langUrl,
-      init_instance_callback(_ed) {
+      init_instance_callback:  (_ed) => {
```

Comment:
Looks like there is an additional space here?

Reply:
Hmm... I wonder why the linter wasn't complaining about that :thinking: Either way, I blame my Mac and it's horrible keyboard that likes to do lots of double spaces :stuck_out_tongue:

Reply:
Fixed. Will look into the potential linter issue separately (likely will be fixed by the eslint plugin upgrades).